contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -602,34 +575,39 @@
         },
       },
       thumbnailEncoding: generateGetterSetter('thumbnail_encoding'),
-      channelQuiz: {
+      completionAndDuration: {
         get() {
-          const options = this.getExtraFieldsValueFromNodes('options') || {};
-          return options.modality === ContentModalities.QUIZ;
-        },
-        set(val) {
-          const options = { modality: val ? ContentModalities.QUIZ : null };
-          this.updateExtraFields({ options });
-        },
-      },
-      // TODO remove eslint disable when `completionCriteria` is utilized
-      /* eslint-disable-next-line kolibri/vue-no-unused-properties */
-      completionCriteria: {
-        get() {
-          const options = this.getExtraFieldsValueFromNodes('options') || {};
-          return options.completion_criteria || {};
+          const { completion_criteria, modality } =
+            this.getExtraFieldsValueFromNodes('options') || {};
+          const suggested_duration_type = this.getExtraFieldsValueFromNodes(
+            'suggested_duration_type'
+          );
+          const suggested_duration = this.getValueFromNodes('suggested_duration');
+          return {
+            suggested_duration,
+            suggested_duration_type,
+            modality,
+            ...(completion_criteria || {}),
+          };
         },
-        set(completion_criteria) {
+        set({ completion_criteria, suggested_duration, suggested_duration_type, modality }) {
           // TODO Remove validation if unnecessary after implementing `completionCriteria`
-          if (validateCompletionCriteria(completion_criteria)) {
-            const options = { completion_criteria };
-            this.updateExtraFields({ options });
-          } else {
-            console.warn('Invalid completion criteria', [...validateCompletionCriteria.errors]);
+          // if (validateCompletionCriteria(completion_criteria)) {
```

Comment:
Might still be good to validate here? I assume from prior conversations that we won't be trying to save invalid completion criteria.